src/deploy/increment-cachebuster.js

```diff
@@ -1,5 +1,5 @@
 const path = require("path");
-require('whatwg-fetch');
+require('node-fetch');
```

Comment:
what does node-fetch do different?
we have been otherwise using watwg-fetch successfully so far